README.md

```diff
@@ -1,27 +1,41 @@
 Cloudinary Android SDK
 ======================
+[![Build Status](https://api.travis-ci.com/cloudinary/cloudinary-android.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/cloudinary_android)
 
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
+## About
+The Cloudinary Android SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
 
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software.
-Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements.
-Images are seamlessly delivered through a fast CDN, and much much more.
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Android SDK Guide](https://cloudinary.com/documentation/android_integration).
 
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
+## Table of Contents
+- [Key Features](#key-features)
+- [Compatibility](#Compatibility)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Uploading Asset](#Uploading)
 
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework.
+## Key Features
+- [Image Transformation](https://cloudinary.com/documentation/android_image_manipulation) and [Video Transformation](https://cloudinary.com/documentation/android_video_manipulation) assets (links to docs).
 
-## Requirements
-The library requires Android version 4.0.3 (Ice Cream Sandwich) and up.
+## Compatibility
+* Minimum Android SDK: Cloudinary requires a minimum API level of 19.
+* Compile Android SDK: Cloudinary requires you to compile against API 31 or later.
 
-## Gradle Integration
+## Installation
+
+### Gradle Integration
 Add the following dependency to your build.gradle:
 
-`implementation group: 'com.cloudinary', name: 'cloudinary-android', version: '1.30.0'`
-## Manual Setup ######################################################################
-Download cloudinary-android-1.30.0.jar from [here](http://central.maven.org/maven2/com/cloudinary/cloudinary-android/1.30.0/cloudinary-android-1.30.0.aar) and cloudinary-core-1.26.0.jar from [here](http://central.maven.org/maven2/com/cloudinary/cloudinary-core/1.26.0/cloudinary-core-1.26.0.jar) and put them in your libs folder.
+`implementation 'com.cloudinary:cloudinary-android:1.30.0'`
+### Manual Setup ######################################################################
```

Comment:
Done
Though I'm afraid people will not understand where the maven installation instructions end and the manual begins since I've swapped the titles into one "More Options"